AmericanMedicalAssociation/ama-style-guide-2
# Footer Updates

## Ticket(s)
**Jira Ticket**
- [EWL-6515 | Homepage - mobile footer logo](https://issues.ama-assn.org/browse/EWL-6515)
- [EWL-6516 | Homepage - mobile footer](https://issues.ama-assn.org/browse/EWL-6516)
- [EWL-6517 | Homepage mobile -footer](https://issues.ama-assn.org/browse/EWL-6517)
- [EWL-6518 | Homepage footer - mobile](https://issues.ama-assn.org/browse/EWL-6518)
- [EWL-6288 | Enhancement: Footer - Instagram Logo Display Issue](https://issues.ama-assn.org/browse/EWL-6288)

## Description
Does the following work on the footer.
- Footer - Instagram logo needs to be swapped out
- Footer - iPhone and Android text should be underlined on default, not just hover (deviates from other link style)
- Footer - AMA logo should be centered in the footer 
- Footer - more padding above and below the social icons 
- Footer - between download/connect / ama careers - should be line here and padding
- Footer - subscribe promo should have subscribe button on the same line as the input

Please reference [Footer mobile zeplin](https://app.zeplin.io/project/59dd1f9c36a8b21d569bb9e8/screen/5b4e3a57729e51c208b6a254) for mobile specific updates.


## To Test
- Pull `bugfix/footer-updates` branch
- Run `gulp serve` in ama-style-guide-2/styleguide to ensure a local version of style guide is running.
- This work affects visuals in D8 and should be tested against it to ensure there are no regressions. Update ama-d8 branch and local provision vars accordingly. 
- Go to any page and confirm requested updates to the footer in linked tickets are present
- Test this in IE 11, Safari, and Firefox to ensure that with updates everything works same as before.

## Visual Regressions
A report is available [here](http://htmlpreview.github.io/?https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/visual-regression-testing-artifact/bugfix/footer-updates/html_report/index.html).


## Relevant Screenshots/GIFs
Screenshot from Drupal
![image](https://user-images.githubusercontent.com/4438120/48023050-5c541a80-e102-11e8-8a7f-57612e46cc80.png)


## Remaining Tasks
N/A


## Additional Notes
N/A
